content/lessons/js2/sublesson/map_reduce_and_filter_functions.mdx

```diff
@@ -1406,7 +1406,8 @@ When the function is running, `this` refers to the object that comes before `.`.
 ```jsx
 Array.prototype.last = function () {
   return this[this.length - 1]
-}[(1, 2, 3)].last() // When the last function is run, 'this' refers to [1,2,3]
+};
```

Comment:
Can you remove the semicolons at the end on this line as well ass line 1419? I just searched through the file, I don't see any javascript code using the semicolon, so it shouldn't be here either

Reply:
Hi Anthony, I posted the reason for semicolon in the Discord bugs & feedback section with the screenshots.

Reply:
the code requires semi-colon here.
A person can try to run the code without semi-colon but rather with just space, both VS CODE and RUNJS will throw errors.

Screenshots with errors
https://media.discordapp.net/attachments/836343487531712512/1009987825606869102/unknown.png?width=972&height=663
https://media.discordapp.net/attachments/836343487531712512/1009987913280393296/unknown.png?width=1440&height=246

Screenshots with output without errors
https://media.discordapp.net/attachments/836343487531712512/1009988225990930604/unknown.png
https://media.discordapp.net/attachments/836343487531712512/1009988226452307978/unknown.png?width=1019&height=663

Reply:
I see. Can you please put this in the PR description? And if there was any other bugs you solved, can you also include it in the PR description too

Reply:
> I see. Can you please put this in the PR description? And if there was any other bugs you solved with this code change, can you also include it in the PR description too

Anthony, description has been updated as requested. Thank you :)

Reply:
@pauly48 approved, but I think you might have forgotten to save the updates you made to the PR description

Reply:
Oh i see you included the image. Please include the explanation as well